apps/store/themes/store/js/categorization.js

```diff
@@ -0,0 +1,261 @@
+$(function() {
+    var updateFilters = function (url){
+        var queryArray = url.split("=")[1].split(",");
+        var queryObjArray = [];
+        if(queryArray.length > 0){
+            for(var index in queryArray){
+                var obj = queryArray[index];
+                var queryObj = {};
+                var queryObjValue = [];
+                queryObj.queryKey = obj.split(":")[0].split('\"').join('');
+                if(obj.indexOf("(") > -1){
+                    var newObj = obj.split(":")[1].split('\"').join('').replace("(", "").replace(")", "");
+                    queryObjValue = newObj.split("OR");
+                } else {
+                    if(obj.indexOf(":") > -1){
+                        queryObjValue.push(obj.split(":")[1].split('\"').join(''));
+                    }
+                }
+                queryObj.queryValue = queryObjValue;
+                queryObjArray.push(queryObj);
+            }
+
+
+            for(var i in queryObjArray){
+                $('#categorization :checkbox').each(function() {
+                    var $this = $(this);
+                    if($this.attr('name') == queryObjArray[i].queryKey){
+                        for(var k in queryObjArray[i].queryValue){
```

Comment:
Please refactor this for loop as follows:

var queryArray = queryObjArray[i].queryValue;
for(var queryKey in queryArray) {

}